docs/appendices/release-notes/unreleased.rst

```diff
@@ -53,6 +53,9 @@ None
 Changes
 =======
 
+- Added statistics and detailed information about the shards located on the
+  node to the ``NodeInfo MXBean`` which is available as an enterprise feature.
```

Comment:
```suggestion
- Added information about the shards located on the node to the ``NodeInfo MXBean`` which is available as an enterprise feature.
```

We aren't really exposing statistics. It's only the number of the number of shard copies and their size and state.
Could you also turn `NodeInfo MXBean` into a link to the relevant section in `monitoring.rst` ?